modules/perception/obstacle/camera/cipv/BUILD

```diff
@@ -0,0 +1,20 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "perception_obstacle_camera_cipv",
```

Comment:
s/perception_obstacle_camera_cipv/camera_cipv/

Reply:
Will update it!